core/src/services/azblob/backend.rs

```diff
@@ -143,6 +150,45 @@ impl AzblobBuilder {
         self
     }
 
+    /// Set server_side_encryption_customer_key of this backend.
+    pub fn encryption_key(&mut self, encryption_key: &str) -> &mut Self {
+        if !encryption_key.is_empty() {
+            self.encryption_key = Some(encryption_key.to_string());
+        }
+
+        self
+    }
+
+    /// Set server_side_encryption_customer_key_sha256 of this backend.
+    pub fn encryption_key_sha256(&mut self, encryption_key_sha256: &str) -> &mut Self {
+        if !encryption_key_sha256.is_empty() {
+            self.encryption_key_sha256 = Some(encryption_key_sha256.to_string());
+        }
+
+        self
+    }
+
+    /// Set server_side_encryption_customer_algorithm of this backend.
```

Comment:
Please add a comment for the valid values.

Reply:
They have now been added.